OpenRA.Mods.Common/Traits/ExternalResourceMultiplier.cs

```diff
@@ -0,0 +1,76 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Mods.Common.Effects;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Gives additional cash when resources are delivered to refineries.")]
+	public class ExternalResourceMultiplierInfo : ConditionalTraitInfo
+	{
+		[FieldLoader.Require]
+		[Desc("Percentage value of the resource to grant as cash.")]
+		public readonly int Modifier;
+
+		public readonly bool ShowTicks = true;
+		public readonly int TickLifetime = 30;
```

Comment:
This is not used, too.

Reply:
These seem to come from Refinery trait, which they seem to be hardcoded to use 30 there too, should i fix it so it uses what defined too.

Reply:
Line 76 of this very file.

Reply:
I fixed it after @abcdefg30 said that.

Reply:
Weird. They were definitely in use last i seen my own code.